auslib/web/admin/swagger/api.yaml

```diff
@@ -254,6 +276,19 @@ parameters:
               minimum: 0
               example: 85142
 
+  sc_signoff_post_body_param:
+    name: sc_post_signoffs_body
+    in: body
+    description: request body for posting a signoff to a scheduled change
+    required: true
+    schema:
+      type: object
+      allOf:
+        - $ref: '#/definitions/CSRFModel'
+        - $ref: '#/definitions/RoleModel'
+      required:
+        - role
```

Comment:
it is already a required field specified in the CSRFModel. here's the [link](https://github.com/aksareen/balrog/blob/ddee69f55db14de816c84d3615426661c3dce846/auslib/web/admin/swagger/api.yaml#L2794) . Role is explicitly required in some APIs and not in others, hence specified on this [line](https://github.com/aksareen/balrog/blob/ddee69f55db14de816c84d3615426661c3dce846/auslib/web/admin/swagger/api.yaml#L290)